src/storage/txn/commands/prewrite.rs

```diff
@@ -618,6 +628,8 @@ impl MutationLock for Mutation {
     }
 }
 
+/// The `MutationLock` variant for pessimistic transactions. The `bool` indicates the
```

Comment:
ditto

Reply:
I agree. I think after documenting both the Trait and the Struct, we can skip the doc for the impl.